stylesheets/_font_stack.scss

```diff
@@ -4,16 +4,23 @@
 // scss-lint:disable NameFormat
```

Comment:
Yes, makes sense.
I'm not sure but if I understand the docs correctly, that disables it for the entire file anyway, independent of the line being at the top or at the bottom. But even if that's the case, it still makes sense to have it closer to what it refers to.